pkgs/tools/typesetting/tex/texlive/combine.nix

```diff
@@ -262,16 +281,6 @@ in (buildEnv {
   # TODO: a context trigger https://www.preining.info/blog/2015/06/debian-tex-live-2015-the-new-layout/
     # http://wiki.contextgarden.net/ConTeXt_Standalone#Unix-like_platforms_.28Linux.2FMacOS_X.2FFreeBSD.2FSolaris.29
 
-    # I would just create links from "$out"/share/{man,info},
-    #   but buildenv has problems with merging symlinks with directories;
-    #   note: it's possible we might need deepen the work-around to man/*.
-  ''
-    for d in {man,info}; do
-      [[ -e "$out/share/texmf/doc/$d" ]] || continue;
-      mkdir -p "$out/share/$d"
-      ln -s -t "$out/share/$d" "$out/share/texmf/doc/$d"/*
```

Comment:
I'm confused by this bit. Is this not taken care of anymore?

Reply:
Nvm. I see, that's done in `${texmf}`.

Reply:
This happens in `texmf` already. I wouldn't do that here, precisely because it would interfere with the man pages in `bin.out`.

Reply:
Oh, you had replied while I was typing!